jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpanContext.java

```diff
@@ -166,14 +151,19 @@ boolean isDebugIdContainerOnly() {
    * debug trace, and the value of header recorded as a span tag to serve as a searchable
    * correlation ID.
    *
-   * @param debugId arbitrary string used as correlation ID
+   * @param debugId arbitrary string used as correlation ID.
+   * @param factory tracing object factory.
    *
    * @return new dummy JaegerSpanContext that serves as a container for debugId only.
    *
    * @see Constants#DEBUG_ID_HEADER_KEY
    */
+  public static JaegerSpanContext withDebugId(String debugId, JaegerObjectFactory factory) {
+    return factory.createSpanContext(0, 0, 0, (byte) 0, Collections.<String, String>emptyMap(), debugId);
+  }
```

Comment:
who is calling it?  I assume codecs only, so same fate as with the other public static methods.

Reply:
OK will remove.